cmd/glasskube/cmd/describe.go

```diff
@@ -45,7 +45,33 @@ var describeCmd = &cobra.Command{
 		for _, ref := range manifest.References {
 			fmt.Printf(" * %v: %v\n", ref.Label, ref.Url)
 		}
+
+		fmt.Printf("\n%v\n", bold("Entrypoints:"))
+		if len(*entrypoints) == 0 {
+			fmt.Fprintln(os.Stderr, " * No Entry Points")
+		} else {
+			for _, i := range *entrypoints {
+				fmt.Fprintf(os.Stderr, " * ")
+				if i.Name != "" {
+					fmt.Fprintf(os.Stderr, "Name: %s, ", i.Name)
+				}
+				if i.ServiceName != "" {
+					fmt.Fprintf(os.Stderr, "ServiceName: %s, ", i.ServiceName)
+				}
+				if i.Port != 0 {
+					fmt.Fprintf(os.Stderr, "Port: %v, ", i.Port)
+				}
+				if i.LocalPort != 0 {
+					fmt.Fprintf(os.Stderr, "LocalPort: %v, ", i.LocalPort)
+				}
+				if i.Scheme != "" {
+					fmt.Fprintf(os.Stderr, "Scheme: %s\n", i.Scheme)
```

Comment:
if `Scheme == ""`, the output will not contain the linebreak at the end. And also the colon would be the last sign of an entrypoint. 

I would suggest to use the `strings.Join` function here. Inside the loop you can hold an array `messageParts []string`, and for each property you can do `messageParts = append(messageParts, fmt.Sprintf("<property-name>: %s", <property-value>))` and at the end of the loop you call the join function on these message parts with the "," as delimiter.